python/lsst/obs/base/_read_curated_calibs.py

```diff
@@ -171,42 +197,77 @@ def read_all(
     it. The detector ID may be retrieved from the DETECTOR entry of that
     metadata.
     """
+    calibration_data = {}
+
     root = os.path.normpath(root)
     dirs = os.listdir(root)  # assumes all directories contain data
     dirs = [d for d in dirs if os.path.isdir(os.path.join(root, d))]
-    data_by_chip = {}
-    name_map = {
-        det.getName().lower(): det.getName() for det in camera
-    }  # we assume the directories have been lowered
-
     if not dirs:
         raise RuntimeError(f"No data found on path {root}")
 
     calib_types = set()
+    detector_map = {
+        det.getName().lower(): det.getName() for det in camera
+    }  # we assume the directories have been lowered
+    filter_map = {
+        filterName.lower(): filterName for filterName in filters
+    }
+
+    paths_to_search: list[Any] = []
```

Comment:
Is it not something like `list[Tuple[str, ...]]` ? `Any` is giving up and saying you can allow anything. In some complex situations that works but in this case it doesn't seem like it's something that we have to be turning off mypy over.